install_config/install/advanced_install.adoc

```diff
@@ -2186,25 +2187,14 @@ $ docker pull docker.io/openshift/origin-ansible:v3.6
 endif::[]
 ----
 
-. The installer system container must be stored in
+. To use the installer image as a system container, it must be stored in
 link:https://access.redhat.com/documentation/en-us/red_hat_satellite/6.2/html/content_management_guide/managing_ostree_content[OSTree]
-instead of defaulting to *docker* daemon storage. Use the Atomic CLI to import
-the installer image from the local *docker* engine to OSTree storage:
+instead of *docker* daemon storage. As the root user, use the Atomic
```

Comment:
atomic install and systemctl and journalctl are generally things you have to run as root. Is it customary to use the root prompt in docs?